### PR TITLE
feat(button-group): add button-group to indigo theme

### PR DESCRIPTION
**Pull request by didimmova**
https://xd.adobe.com/view/7f7fc9aa-8ce4-4e07-a6d6-c8efe9ebce98-2a2c/

### Additional information (check all that apply):
 - [ ] Bug fix
 - [x] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [ ] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss [hunk 1]
```diff
@@ -211,3 +211,54 @@ $_bootstrap-button-group: extend(
...
+    $_indigo-shape-button-group,
+    $_indigo-elevation-button-group,
+    (
+        variant: 'indigo-design',
+
+        item-background: (
+            igx-color: ('primary', 500),
+        ),
+
+        item-text-color: #fff,
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss:225]: I'd rather that be the contrast color of `primary 500`.

File: projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss [hunk 2]
```diff
@@ -211,3 +211,54 @@ $_bootstrap-button-group: extend(
...
+
+        item-border-color: (
+            igx-color: ('primary', 400),
+        ),
+
+        item-hover-background: (
+            igx-color: ('primary', 400),
+        ),
+
+        item-hover-text-color: #fff,
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss:235]: Same here but for `primary 400`.

File: projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss [hunk 3]
```diff
@@ -211,3 +211,54 @@ $_bootstrap-button-group: extend(
...
+
+        item-selected-background: (
+            igx-color: ('primary', 400)
+        ),
+
+        item-selected-text-color: #fff,
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss:241]: And here.

File: projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss [hunk 4]
```diff
@@ -211,3 +211,54 @@ $_bootstrap-button-group: extend(
...
+            rgba: .4
+        ),
+
+        disabled-text-color: #fff,
+
+        item-disabled-border: (
+            igx-color: ('primary', 400),
+            rgba: .4
+        )
+    )
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss:262]: I am not really sure about those disabled colors. Especially the bright white that's used for text. Plus, the button group and the button differ too much in that respect.

Comment (didimmova) [reply projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss:262]: @StefanIvanov

Comment (StefanIvanov) [reply projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss:268]:
```suggestion
        disabled-background-color: (
            igx-color: ('grays', 100)
        ),

        disabled-text-color: (
            igx-color: ('grays', 400)
        ),

        item-disabled-border: (
            igx-color: ('grays', 100)
        ),
    )
```

Comment (simeonoff) [reply projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss]: These look a lot better.

**Pull request merged**